packages/ses/test/test-anticipate-async-iterator-helpers-shimmed.js

```diff
@@ -0,0 +1,38 @@
+// KLUDGE HAZARD The core-js shims are written as sloppy code
+// and so introduce sloppy functions.
```

Comment:
Afaik, core-js is compatible with strict mode, or at least most parts of it are. I actually never thought about CommonJS support in ES Modules loaded by node as being able to introduce sloppy functions (besides through dynamic eval).

Wondering if we could force the tests to be run with a node CommonJS module loader that forces `'use strict'`

Reply:
> Wondering if we could force the tests to be run with a node CommonJS module loader that forces 'use strict'

@kriskowal Is there such a thing?

Reply:
Filed https://github.com/endojs/endo/issues/1669 on this. If this PR becomes otherwise ready to merge, I'll let that issue speak for this conversation. So resolving here.